src/System.Management.Automation/engine/CommandDiscovery.cs

```diff
@@ -1281,27 +1281,6 @@ internal void UnregisterLookupCommandInfoAction(string currentAction, string com
         private HashSet<string> _activeCommandNotFound = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Command Discovery tests.Native command discovery.Can discover a native command with extension on Windows
```powershell
Expected regular expression 'ping.exe' to match $null, but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\CommandDiscovery.Tests.ps1: line 227
227:             (Get-Command -Name "pingG.exe" -CommandType Application).Name | Should -Match "ping.exe"
```